programs/leap-cli11/README.md

```diff
@@ -0,0 +1,26 @@
+
+## instructions for building custom leap-cli11 library
+
+leap-cli11 interface only library created in order to simplify integration of modified version of command line parsing library cli11
+
+to update included in this library include/cli11/CLI11.hpp file it needs to be (re)geenerated from repository containing forked/modified version of it, in particular:
+
+
+```bash
+git clone git@github.com:AntelopeIO/CLI11.git
```

Comment:
since it's a public repo I'd recommend changing to https://

Reply:
Changed, ty!